### PR TITLE
Dawn landing page

### PR DESCRIPTION
**Pull request by dawnaldrich**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,5 +31,17 @@
...
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nourimeals/website.git"
+  },
+  "bugs": {
+    "url": "https://github.com/nourimeals/website/issues"
+  },
+  "homepage": "https://github.com/nourimeals/website#readme"
```
Comment (ambergkim) [inline package.json]: Is homepage supposed to direct to our read me or is it meant to direct people to the deployed version?

File: src/components/header/header.scss [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+    //   position: fixed;
+    //   left: 45vw;
+    // }
+
+    // @media only screen and (min-width: 1000px) {
+    //   width: 150px;
+    //   position: fixed;
+    //   left: 45vw;
+    // }
+
```
Comment (ambergkim) [inline src/components/header/header.scss]: Commented code should be removed.

File: src/components/header/header.scss [hunk 3]
```diff
@@ -1,17 +1,34 @@
 .header-nav {
-  text-align: center;
-  padding-top: 20px;
   position: fixed;
-  left: 35vw;
+  top: 0;
+  display: flex;
+  // background-color: #ffffffd8;
```
Comment (ambergkim) [inline src/components/header/header.scss]: Commented code should be removed.

File: src/components/header/header.scss [hunk 4]
```diff
@@ -1,17 +1,34 @@
...
-      width: 150px;
-      position: fixed;
-      left: 45vw;
-    }
+#logo {
+  align-self: center;
+  width: 100px;
+  margin: 0 auto;
+  padding: 5px;
+}
```
Comment (ambergkim) [inline src/components/header/header.scss]: Scss has nesting so this logo should be nested inside .header-nav

File: src/components/home/home.scss [hunk 5]
```diff
@@ -2,58 +2,136 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: $bg-img no-repeat center center;
   background-size: cover;
+  width: 100%;
+  height: 100%;
 
   .cta-block {
+    // width: 400px;
```
Comment (ambergkim) [inline src/components/home/home.scss]: Remove

File: src/components/home/home.scss [hunk 6]
```diff
@@ -2,58 +2,136 @@
...
+        padding-top:50px;
+        font-size: 1.8rem;
+        font-weight:300;
+      }
 
-    h1 {
-      font-size: 70px;
+      p {
+        font-size: 1.1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (ambergkim) [inline src/components/home/home.scss]: We need to have the correct fonts. I tagged you in the Trello card that has them. :)

File: src/components/home/home.scss [hunk 7]
```diff
@@ -2,58 +2,136 @@
...
 
-      @media only screen and (min-width: 700px) {
-        font-size: 30px;
+      .cta-btn {
+        background-color: #FA5457;
+        width: 100%;
+        color: white;
+        font-weight: 100;
+        font-size: 1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (ambergkim) [inline src/components/home/home.scss]: fonts also

File: src/components/home/home.scss [hunk 8]
```diff
@@ -2,58 +2,136 @@
...
-      font-size: 70px;
+      p {
+        font-size: 1.1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+      }
 
-      @media only screen and (min-width: 700px) {
-        font-size: 30px;
+      .cta-btn {
+        background-color: #FA5457;
```
Comment (ambergkim) [inline src/components/home/home.scss]: Let's add use the variables for colors at the top so they are easy to update altogether.

File: src/components/home/home.scss [hunk 9]
```diff
@@ -2,58 +2,136 @@
...
+        text-transform: uppercase;
+        padding: 15px;
+        border: 1px solid #000;
+        margin-bottom: 70px;
       }
+
+
+   
+
+    // h1 {
```
Comment (ambergkim) [inline src/components/home/home.scss]: Remove these, too

File: src/components/home/home.scss [hunk 10]
```diff
@@ -2,58 +2,136 @@
...
+    height: 100%;
+
+    .cta-block.clear {
+      clear:both;
     }
 
-    p {
-      font-size: 60px;
+    .cta-block {
+      // width: 400px;
```
Comment (ambergkim) [inline src/components/home/home.scss]: ditto

File: src/components/home/home.scss [hunk 11]
```diff
@@ -2,58 +2,136 @@
...
+      box-sizing: border-box;
+      background-color: #ffffffd8;
+      height: 50%;
+      max-width: 60%;
+      padding: 10px;
+      // background-color: green;
```
Comment (ambergkim) [inline src/components/home/home.scss]: ditto

File: src/components/home/home.scss [hunk 12]
```diff
@@ -2,58 +2,136 @@
...
+    .cta-block.clear {
+      clear:both;
     }
 
-    p {
-      font-size: 60px;
+    .cta-block {
+      // width: 400px;
+      box-sizing: border-box;
+      background-color: #ffffffd8;
```
Comment (ambergkim) [inline src/components/home/home.scss]: Use the variables or add them to the variables if it's not there.

File: src/components/home/home.scss [hunk 13]
```diff
@@ -2,58 +2,136 @@
...
+      h1 {
+        font-size: 1.8rem;
+        font-weight:300;
+      }
 
-      @media only screen and (min-width: 700px) {
-        font-size: 20px;
+      p {
+        font-size: 1.1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (ambergkim) [inline src/components/home/home.scss]: update the right font

File: src/components/home/home.scss [hunk 14]
```diff
@@ -2,58 +2,136 @@
...
-      font-size: 70px;
-      text-transform: uppercase;
-      padding: 20px;
-
-      @media only screen and (min-width: 700px) {
-        font-size: 20px;
-        padding: 10px;
-        width: 300px;
+      .cta-btn {
+        background-color: #FA5457;
```
Comment (ambergkim) [inline src/components/home/home.scss]: Color also

File: src/components/home/home.scss [hunk 15]
```diff
@@ -2,58 +2,136 @@
...
+        width: 50%;
+        color: white;
+        font-weight: 100;
+        font-size: 1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (ambergkim) [inline src/components/home/home.scss]: font also

File: src/style/main.scss [hunk 16]
```diff
@@ -1,3 +1,5 @@
+
 .white-overlay {
   background-color: #ffffffd8;
```
Comment (ambergkim) [inline src/style/main.scss]: Variables also

File: package.json (hunk 1 above)
Comment (dawnaldrich) [reply package.json]: Do you have a preference? This wasn't something I setup but will update on request.

Comment (ambergkim) [reply package.json]: @dawnaldrich We can leave it as-is for now.

Comment (ambergkim) [reply package.json]: We can leave for now.

File: src/components/home/home.jsx [hunk 17]
```diff
@@ -30,21 +30,24 @@ class HomePage extends React.Component {
 
   render() {
     return <React.Fragment>
+
```
Comment (ambergkim) [inline src/components/home/home.jsx]: FYI. Careful of whitespace changes. Just so there's less stuff for reviewers to look at.

**Comment by ambergkim** [comment]
@dawnaldrich @ryanscottjohnson Almost ready. Just need to rebase and resolve the merge conflicts.

**Pull request merged**